guides/doc-Provisioning_Hosts/master.adoc

```diff
@@ -26,7 +26,9 @@ include::common/assembly_using-infoblox-as-dhcp-and-dns-providers.adoc[leveloffs
 
 include::common/assembly_using-pxe-to-provision-hosts.adoc[leveloffset=+1]
 
+ifndef::satellite[]
```

Comment:
I'm not sure that this is correct. Is there a connection between booting with installation media and iPXE boot?

Reply:
The connection was supportability, but let's discuss on Slack